src/editors/utils/formatLibraryImgRequest.ts

```diff
@@ -0,0 +1,134 @@
+import StrictDict from './StrictDict';
+
+/**
+ * A dictionary that maps file extensions to their corresponding MIME types for images.
+ *
+ * @example
+ * acceptedImgMimeTypes.gif // "image/gif"
+ */
+
+const acceptedImgMimeTypes = StrictDict({
+  gif: 'image/gif',
+  jpg: 'image/jpg',
+  jpeg: 'image/jpeg',
+  png: 'image/png',
+  tif: 'image/tiff',
+  tiff: 'image/tiff',
+  ico: 'image/x-icon',
+});
+
+type TinyMCEImageData = {
+  displayName: string,
+  contentType: string,
+  url: string,
+  externalUrl: string,
+  portableUrl: string,
+  thumbnail: string,
+  id: string,
+  locked: boolean,
+};
+
+export type LibraryAssetResponse = {
```

Comment:
It would be good to move this type into `library-authoring/data.api.ts` and then share it between this file and this existing function:

https://github.com/openedx/frontend-app-authoring/blob/e118eb5971f0d52854aa5e545b4bf987ec7adc8d/src/library-authoring/data/api.ts#L442